windows/variables.props.dev

```diff
@@ -5,10 +5,12 @@
     <!-- Change the following paths so they are correct on your machine -->
     <!-- Do not modify anything before this line -->
     <MKLDIR>C:\Program Files (x86)\IntelSWTools\compilers_and_libraries\windows\mkl\</MKLDIR>
-    <OPENBLASDIR>C:\Users\Yenda\Downloads\kaldi-svn\tools\OpenBLAS-v0.2.14-Win64-int32</OPENBLASDIR>
-    <OPENFST>C:\Users\jtrmal\Documents\openfst\</OPENFST>
-    <OPENFSTLIB>C:\Users\jtrmal\Documents\openfst\build64</OPENFSTLIB>
+    <OPENBLASDIR>W:\kaldi-build\kaldi\tools\OpenBLAS-v0.2.19-Win64-int32</OPENBLASDIR>
```

Comment:
Hi, could you please either change these paths back or make sure the readmes are also corrected? I'm not asking of vanity, just wanna keep the docs in sync, people get confused with the simplest disparities often...

Reply:
Not a problem, and completely understandable.  I've made those changes.